### ISSUE TITLE
`ReferenceFE` factory function

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#385]
It would be nice to have a factory function with several kw-args for the creation of `ReferenceFE` instances. Like we have for `FESpace`.

Example:
```julia
reffe = ReferenceFE(dofbasis=:Lagrangian,order=1,valuetype=Float64)
V = FESpace(reffe = reffe, model = model, conformity = :L2)
```
This would be nice for several reasons. One of them is that it reduces the number of arguments in `FESpace` function.

Any volunteer ?

**Comment by fverdugo** [comment]
BTW, `FESpace` already accepts an instance of `ReferenceFE`. E.g., this is currently working (for 3D obviously):

```julia
V = FESpace(reffe = HEX8, model = model, conformity = :L2)
```

**Comment by fverdugo** [comment]
This was resolved in Gridap 0.15.
